test/runtests.jl

```diff
@@ -188,6 +188,16 @@ end
             end
         end
     end
+
+    srcdir = joinpath(@__DIR__, "BustedPackage", "src")
+    clean_folder(srcdir)
+    cmdstr = "using BustedPackage; BustedPackage.greet()"
+    cd("BustedPackage") do
+        run(`$(Base.julia_cmd()) --startup-file=no --code-coverage=user --project -e $cmdstr`)
+    end
+    bustedfile = joinpath(srcdir, "parseerr.jl")
+    @test_throws Base.Meta.ParseError("parsing error in $bustedfile:7: space before \"[\" not allowed in \"i [\"") process_file(bustedfile, srcdir)
```

Comment:
Annoyingly the message differs between older Julia versions and 1.5 (or so), where the string " at none:4" is added to the message. 
```suggestion
    msg = "parsing error in $bustedfile:7: space before \"[\" not allowed in \"i [\""
    if VERSION >= v"1.5"
        msg *= " at none:4"
    end
    @test_throws Base.Meta.ParseError(msg) process_file(bustedfile, srcdir)
```